frontend/src/features/admin-form/settings/components/EmailFormSection.tsx

```diff
@@ -48,22 +79,47 @@ export const EmailFormSection = ({
     [initialEmailSet, mutateFormEmails],
   )
 
-  useEffect(() => reset({ emails: initialEmails }), [initialEmails, reset])
+  useEffect(() => reset({ emails: settings.emails }), [settings.emails, reset])
+
+  const emailModeDescription = `Add at least **2 recipients** to prevent loss of response.`
+  const storageModeDescription = `FormSG securely stores responses in an encrypted format and does not retain any associated emails.`
 
   return (
     <>
-      <MRFAdvertisingInfobox />
+      {isFormPublic ? (
+        <InlineMessage marginBottom="16px">
+          To change admin email recipients, close your form to new responses.
+        </InlineMessage>
+      ) : isEmailInputDisabled ? (
+        <InlineMessage variant="error">
+          To enable email notifications, please remove Payments fields from your
+          form.
+        </InlineMessage>
+      ) : settings && settings.responseMode === FormResponseMode.Encrypt ? (
```

Comment:
/nit `settings?.responseMode === FormResponseMode.Encrypt`

Reply:
Thanks for the catch!